pipeline-runner/src/data-ingest/5_Upload-to-aws.r

```diff
@@ -223,6 +212,17 @@ task <- function(input, pipeline_config) {
   cluster_env <- pipeline_config$cluster_env
   print(sprintf("Experiment ID: %s uploaded to %s.", experiment_id, cluster_env))
 
+  send_dynamodb_item_to_api(pipeline_config,
```

Comment:
<Rant not strictly related to this PR>
I think that we are leaking too much information from the API to the pipeline: dynamodb and table names should be something that probably belong only to the API.  But changing this is out of the scope of this PR.
</Rant not strictly related to this PR>

Reply:
We should, could you create a ticket for it?